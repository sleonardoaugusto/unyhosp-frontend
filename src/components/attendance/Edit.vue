<template>
  <div>
    <h1>Edição de atendimento</h1>
  </div>
</template>

<script>
export default {
  props: ['attendanceId'],
  name: 'Edit'
}
</script>

<style scoped>

</style>