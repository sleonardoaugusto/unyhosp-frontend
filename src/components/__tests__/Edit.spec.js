import { shallowMount } from '@vue/test-utils'
import Edit from '@/components/attendance/Edit'

describe('Edit.vue', () => {
  let wrapper

  wrapper = shallowMount(Edit)

  it('component is rendering properly', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})