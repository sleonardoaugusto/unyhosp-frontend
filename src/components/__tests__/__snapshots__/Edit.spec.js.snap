// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Edit.vue component is rendering properly 1`] = `
<div class="d-flex flex-wrap justify-content-center align-items-center flex-column">
  <div class="col-md-6 mb-3">
    <div class="row">
      <h2 class="type-h4 type-light-green">Editar atendimento</h2>
    </div>
  </div>
  <div class="col-md-6">
    <div class="row d-flex flex-column mb-3"><label for="PacientName">Nome:</label> <input type="text" id="PacientName"></div>
    <div class="row d-flex flex-column mb-3"><label for="Entry">Dias de UTI:</label> <input type="text" id="Entry"></div>
    <div class="row d-flex flex-column mb-3"><label for="Age">Idade:</label> <input type="text" id="Age"></div>
    <div class="row d-flex flex-column mb-3"><label for="HMA">HMA</label> <input type="text" id="HMA"></div>
    <div class="row d-flex flex-column mb-3"><label for="EntryReason">Motivo da Admissão:</label> <input type="text" id="EntryReason"></div>
    <div class="row d-flex flex-column mb-3"><label for="Comorbidity">Comorbidades</label> <input type="text" id="Comorbidity"></div>
    <div class="row d-flex flex-column mb-3"><label for="ATB">ATB:</label> <input type="text" id="ATB"></div>
    <div class="row d-flex flex-column mb-3"><label for="HD">HD:</label> <input type="text" id="HD"></div>
    <div class="row d-flex flex-column mb-3"><label for="MedicinesInUse">Medicamentos em uso:</label> <input type="text" id="MedicinesInUse"></div>
    <div class="row d-flex flex-column mb-3"><label for="PreviousPathologies">Patologias Prévias:</label> <input type="text" id="PreviousPathologies"></div>
    <div class="row d-flex flex-column mb-3"><label for="Allergies">Alergias:</label> <input type="text" id="Allergies"></div>
    <div class="row d-flex flex-column mb-3"><label for="Cultures">Culturas:</label> <input type="text" id="Cultures"></div>
    <div class="row d-flex flex-column mb-3"><label for="VasoactiveDrugs">Drogas Vasoativas:</label> <input type="text" id="VasoactiveDrugs"></div>
    <div class="row d-flex flex-column mb-3"><label for="Sedation">Sedação:</label> <input type="text" id="Sedation"></div>
    <div class="row d-flex flex-column mb-3"><label for="VenousAccess">Acesso Venoso:</label> <input type="text" id="VenousAccess"></div>
    <div class="row d-flex flex-column mb-3"><label for="Diet">Dieta:</label> <input type="text" id="Diet"></div>
    <div class="row d-flex flex-column mb-3"><label for="ProbesAndDrains">Sondas e Drenos:</label> <input type="text" id="ProbesAndDrains"></div>
    <div class="row d-flex flex-column mb-3"><label for="Ventilation">Ventilação:</label> <input type="text" id="Ventilation"></div>
    <div class="row d-flex flex-column mb-3"><label for="Complications">Complicações:</label> <input type="text" id="Complications"></div>
    <div class="row d-flex flex-column mb-3"><label for="TherapeuticPlan">Plano Terapêutico</label> <input type="text" id="TherapeuticPlan"></div>
  </div>
</div>
`;
